README.md

```diff
@@ -81,9 +81,21 @@ Run `carthage update` to build the framework and drag the built `WeScan.framewor
 
 Just download the project, and drag and drop the "WeScan" folder in your project.
 
+
+<p align="center">
+    <img width="900px" src="Assets/project.png">
+</p>
+
+
+Simply Add the WeScan FrameWork into project `Embedded Binaries` and `Linked Frameworks and Libraries`.
```

Comment:
This sentence is not quite correct. Here is a better version:
>  Simply Add the WeScan framework into the project's `Embedded Binaries` and `Linked Frameworks and Libraries`.

Reply:
Minor correction: I don't think "Add" should be capitalized in the corrected sentence.

Reply:
You're right @justJS!